errors/invalid-getstaticprops-value.md

```diff
@@ -9,7 +9,7 @@ In one of the page's `getStaticProps` the return value had the incorrect shape.
 Make sure to return the following shape from `getStaticProps`:
 
 ```js
-export async function getStaticProps() {
+export async function getStaticProps({ params }) {
```

Comment:
```suggestion
export async function getStaticProps(ctx: {
  params: ParsedUrlQuery | undefined
  preview?: boolean
  previewData?: any
}) {
```
Do we want to match the [context type](https://github.com/zeit/next.js/blob/5a828125682b86c34636a07203d505ad358fb38b/packages/next/types/index.d.ts#L67-L71) we provide?

Reply:
@ijjk Thanks! I forgot about adding types and that `preview` / `previewData` got added recently.